pkgs/development/python-modules/python3-application/default.nix

```
@@ -0,0 +1,49 @@
+{ lib, isPy3k, buildPythonPackage, fetchFromGitHub, fetchpatch, zope_interface, twisted }:
+
+buildPythonPackage rec {
+  pname = "python3-application";
+  version = "3.0.3";
+
+  disabled = !isPy3k;
+
+  owner = "AGProjects";
+
+  src = fetchFromGitHub {
+    inherit owner;
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-oscUI/Ag/UXmAi/LN1pPTdyqQe9aAfeQzhKFxaTmW3A=";
+  };
+
+  patches = [
+    # Apply bugfix commit that is not yet part of a release
+    (fetchpatch {
+      name = "fix-time-import.patch";
+      url = "https://github.com/${owner}/${pname}/commit/695f7d769e69c84e065872ffb403157d0af282fd.patch";
```

eventually the patch will be gone, and then we will only need owner once. So i would just inline it for now

```suggestion
  src = fetchFromGitHub {
    owner = "AGProjects";
    repo = pname;
    rev = version;
    sha256 = "sha256-oscUI/Ag/UXmAi/LN1pPTdyqQe9aAfeQzhKFxaTmW3A=";
  };

  patches = [
    # Apply bugfix commit that is not yet part of a release
    (fetchpatch {
      name = "fix-time-import.patch";
      url = "https://github.com/AGProjects/${pname}/commit/695f7d769e69c84e065872ffb403157d0af282fd.patch";
```

---

Makes sense. Done.